### PR TITLE
chore: tweaks to styling pages

### PR DESCRIPTION
**Pull request by gentaDemnushaj**
ATTENTION! NEW DOCS STRUCTURE
FOR THE NEXT THREE WEEKS, ONLY BRANCH FROM new-docs-structure
OTHERWISE, WAIT UNTIL THE NEW STRUCTURE HAS BEEN RELEASED

  - Please check the [internal contributions guide](https://www.notion.so/genesisglobal/Contributing-new-documentation-75953fb245f246ff872789035451a0c4) for details
  
__________

Thank you for contributing to the documentation.

Do the changes you have made apply to both Current and Previous versions?

Have you done a trial build to check all new or changed links?

Is there anything else you would like us to know?

**This week's exciting advice from the style guide**

- Write your headings in sentence case:

  - A good example
    This is a correct heading.
  - A Bad Example
    This is not a correct heading.

### CODE REVIEW

File: docs/001_develop/03_client-capabilities/019_styling/01_extending-genesis-components/02_overriding-default-styles.mdx [hunk 1]
```diff
@@ -148,31 +166,49 @@ import { ElementStyles, PropertyStyleSheetBehavior } from '@genesis/web-core';
  * @public
  */
 export function appearanceBehavior(value: string, styles: ElementStyles) {
-  return new PropertyStyleSheetBehavior('appearance', value, styles);
+  return new PropertyStyleSheetBehavior("appearance", value, styles);
```
Comment (MrBrunoWolff) [inline docs/001_develop/03_client-capabilities/019_styling/01_extending-genesis-components/02_overriding-default-styles.mdx:169]: nitpick: this one should be ok to stay as it is?

File: docs/001_develop/03_client-capabilities/019_styling/01_extending-genesis-components/02_overriding-default-styles.mdx [hunk 2]
```diff
@@ -45,15 +44,19 @@ Let's override structural properties defined in the `baseStyles` by adding or mo
 ##### Custom Base Styles
 
 ```typescript
-import { css } from '@genesislcap/web-core';
```
Comment (MrBrunoWolff) [inline docs/001_develop/03_client-capabilities/019_styling/01_extending-genesis-components/02_overriding-default-styles.mdx:48]: question: are we moving all examples to "double quotes symbol"?

Comment (gentaDemnushaj) [reply docs/001_develop/03_client-capabilities/019_styling/01_extending-genesis-components/02_overriding-default-styles.mdx:48]: i did format document and it must have switched it

**Review by matteematt — APPROVED** [review]
As discussed more to do in future, but defo good enough for now

**Pull request merged**